sopel/config/core_section.py

```diff
@@ -178,6 +178,9 @@ def homedir(self):
     It is a regular expression (so the default, ``\.``, means commands start
     with a period), though using capturing groups will create problems."""
 
+    reply_errors = ValidatedAttribute('reply_errors',bool,default=False)
```

Comment:
As much as I don't like it either, the default for this should be `True`. I only say this to stay consistent with existing configurations.